packages/fxa-settings/README.md

```diff
@@ -51,6 +51,23 @@ FxA Settings communicates primarily with the FxA GraphQL API through use of [Apo
 
 While most API calls can be performed with GQL, there are a few calls that must directly communicate with the auth server (for now) for security reasons. To make REST API calls to the auth server, use the `useAuth` and `useAwait` hooks.
 
+### Error handling
+
+All errors that can occur as a result of a network action should be handled. As a general rule of thumb validation errors should be displayed the user while all others should be reported to Sentry unless they are know to occur and intentionally ignored.
+
+When performing a GraphQL mutation, use our custom [`useMutation`](./lib/hooks.tsx) hook and set the `onError` property to handle GraphQL Errors (such as validations). The hook will automatically send any network-level errors to Sentry and return an `ApolloError` for you to access GQL errors and render them in the UI or an AlertBar. Example:
+
+```tsx
+const [destroySession, { data }] = useMutation(DESTROY_SESSION_MUTATION, {
+  onError: (error) => {
+    // The useAlertBar hook's `error` method can take an ApolloError object
+    // and conditionally render GQL errors if they're present, or fall back
+    // to the error string you provided.
```

Comment:
Oh sweet! I was going to ask if we wanted to display the error to the user but the alert bar is smart.